### PR TITLE
Update ChangeLog with modifications since last release.

### LINKED ISSUE #340: Update ChangeLog with changes since last release
Last release was Jan 2022.

### PR DESCRIPTION
**Pull request by sobolevnrm**
Fixes #340.

### CODE REVIEW

File: docs/source/releases.rst [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+Fixes
+=====
+
+* Fix problems with protonated terminal residue naming (`#301 <https://github.com/Electrostatics/pdb2pqr/pull/301>_`)
+* Fix problem with terminal histidine protonation (`#301 <https://github.com/Electrostatics/pdb2pqr/pull/301>_`)
+
+Changes
+=======
+
+* Update Python version support for compatibility with PROPKA (`#342 <https://github.com/Electrostatics/pdb2pqr/issues/342>_`)
```
Comment (orbeckst) [inline docs/source/releases.rst:18]: I'd explicitly state the currently supported Python version.

Comment (orbeckst) [reply docs/source/releases.rst:18]:
```suggestion
* Update Python version support for compatibility with PROPKA (`#342 <https://github.com/Electrostatics/pdb2pqr/issues/342>_`);
  minimal supported Python is 3.8, highest tested is 3.11
```

**Review by orbeckst — APPROVED** [review]
I suggest to be explicit with the supported Python versions. You can accept my suggestion if you like. Otherwise good.

**Comment by sobolevnrm** [comment]
Thank you, @orbeckst !

**Pull request merged**